.github/workflows/e2e-ci.yaml

```diff
@@ -124,204 +148,244 @@ jobs:
+          FACADE_SERVER_URL: ${{ secrets.FUNCTION_FACADE_SERVER_URL }}
+          FACADE_AUTH_CODE: ${{ secrets.FUNCTION_FACADE_AUTH_CODE }}
+        with:
+          deployment_file_name: '${{ secrets.DEPLOYMENT_FILE_NAME }}'
+          architecture: '${{ secrets.ARCHITECTURE }}'
+          iot_edge_deployment_id: '${{ secrets.IOT_EDGE_DEPLOYMENT_ID }}'
 
-      # Replacement need to occur to be compliant with the IoT Edge tool, we should remove in the future.
-      - run: |
-          $LBS_IMAGE_TAG=$(az acr repository show-tags --username ${{ env.CONTAINER_REGISTRY_USERNAME }} --password ${{ env.CONTAINER_REGISTRY_PASSWORD }} --name ${{ env.CONTAINER_REGISTRY_ADDRESS }} --repository lorabasicsstation --orderby time_desc -o json | ConvertFrom-Json)[0]
+# Deploy IoT Edge solution to EFLOW gateway
+  deploy_eflow_gw_iot_edge:
+    environment:
+      name: CI_EFLOW_AZURE_VM
+      url: https://github.com/Azure/iotedge-lorawan-starterkit/runs/${{ github.run_id }}?check_suite_focus=true
+    needs:
+    - env_var
+    - build_push_docker_images
+    if: needs.env_var.outputs.RunE2ETestsOnly != 'true'
+    runs-on: ubuntu-latest
+    name: Deploy IoT Edge on EFLOW VM gateway
+    env:
+      DEVICE_ID: ${{ secrets.DEVICE_ID }}
+      NET_SRV_LOG_TO_UDP_ADDRESS: ${{ secrets.LOG_TO_UDP_ADDRESS }}
+      IOTHUB_CONNECTION_STRING: ${{ secrets.IOTHUB_OWNER_CONNECTION_STRING }}
+      FACADE_SERVER_URL: ${{ secrets.FUNCTION_FACADE_SERVER_URL }}
+      FACADE_AUTH_CODE: ${{ secrets.FUNCTION_FACADE_AUTH_CODE }}
+      EDGE_AGENT_VERSION: 1.1.8
+      EDGE_HUB_VERSION: 1.1.8
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout current branch
 
-          $LBS_IMAGE_TAG=$LBS_IMAGE_TAG.replace("-${{ matrix.architecture }}",'')
-          echo $LBS_IMAGE_TAG
-          echo "::set-env name=LBS_VERSION::$LBS_IMAGE_TAG"
-        shell: pwsh
-        name: Compute latest LBS build
+      - name: Iot Edge Push Solution
+        uses: ./.github/actions/deployedge
+        id: deploy_edge
         env:
-          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
-
-      - name: Iot Edge Deploy
-        uses: ./devops
-        id: edgedev_deploy
+          IOTHUB_CONNECTION_STRING: ${{ secrets.IOTHUB_OWNER_CONNECTION_STRING }}
+          DEVICE_ID: ${{ secrets.DEVICE_ID }}
+          IOT_EDGE_DEPLOYMENT_ID: ${{ secrets.IOT_EDGE_DEPLOYMENT_ID }}
         with:
-          command: deploy
-          folder_path: 'LoRaEngine'
-          file_name: 'deployment.lbs.template.json'
-          target_platform: ${{ matrix.architecture }}
+          deployment_file_name: '${{ secrets.DEPLOYMENT_FILE_NAME }}'
+          architecture: '${{ secrets.ARCHITECTURE }}'
+          iot_edge_deployment_id: '${{ secrets.IOT_EDGE_DEPLOYMENT_ID }}'
 
+  # Deploy IoT Edge solution to standalone Concentrator
+  deploy_eflow_concentrator_iot_edge:
+    needs:
+    - env_var
+    - build_push_docker_images
+    environment:
+      name: CI_CONCENTRATOR_EFLOW_ARM32
```

Comment:
Isn't this name misleading? It's not EFLOW_ARM32, it's a RPi no?

Reply:
It's a concentrator running on arm32 used to target messages to eflow. This is why the name. I could change it if you don't like it

Reply:
It's fine, I misread 👍